src/main/java/com/iridium/iridiumskyblock/configs/Configuration.java

```diff
@@ -56,6 +56,7 @@ public Configuration() {
     public boolean clearEnderChestOnRegen = false;
     public boolean allowPvPOnIslands = false;
     public boolean islandCreateOnJoin = false;
+    public boolean islandCreationCost = false;
```

Comment:
I feel like this is going to confuse things even more. I understand there are alot of people asking this in the Discord, but if we want to remove the creation cost, I feel setting the default cost to 0 for everything is a better aproach

Reply:
this was more for the fact that we could allow pricing for creation *and* regen to be different (as in, when you create the island, it doesn't cost anything, but when you regen, it does) - we could change it to be free but honestly i dont care about those people lol